package.json

```diff
@@ -109,10 +109,12 @@
     "lib": "./lib"
   },
   "main": "./lib/index",
+  "types": "./lib/index.d.ts",
   "engines": {
     "node": ">=0.10"
   },
   "dependencies": {
+    "@types/node": "^7.0.5",
```

Comment:
Is this used? I don't see it being used anywhere. Alternatively should it be a dev dependency?